tools/profile.py

```diff
@@ -214,10 +218,22 @@ def positive_nonzero_int(val):
 bpf_text = bpf_text.replace('USER_STACK_GET', user_stack_get)
 bpf_text = bpf_text.replace('KERNEL_STACK_GET', kernel_stack_get)
 
+sample_freq = 0
+sample_period = 0
+if args.frequency:
+    sample_freq = args.frequency
+elif args.count:
+    sample_period = args.count
+else:
```

Comment:
If both frequency and count are specified, do you know how perf handle it?

Reply:
The `argparser` makes sure that user can only specify at most one (by using `add_mutually_exclusive_group`). We also have check in `libbpf` (in `bpf_attach_perf_event`) to check only one is specified.